packages/x-components/src/components/column-picker/base-column-picker-dropdown.vue

```diff
@@ -1,33 +1,31 @@
 <template>
-  <div class="x-column-picker-dropdown">
-    <BaseDropdown
-      @change="emitEvent"
-      :value="selectedColumn"
-      :items="columns"
-      :animation="animation"
-    >
-      <template v-if="$scopedSlots['toggle']" #toggle="{ item, isOpen }">
-        <!--
+  <BaseDropdown
+    @change="emitEvent"
+    :value="selectedColumns"
+    :items="columns"
+    :animation="animation"
+  >
+    <template v-if="$scopedSlots.toggle" #toggle="{ item, isOpen }">
+      <!--
            @slot From `BaseDropdown` component: Used to render the contents of the dropdown toggle
            button. If not provided, it uses the `item` slot as fallback.
-               @binding {string|number|Identifiable} item - The item data to render.
```

Comment:
- [ ] Add again the spaces

Reply:
Let's ask in slack, because I remember someone telling me to remove those spaces in other PRs